src/main/scala/castalia/Delay.scala

```diff
@@ -19,9 +20,143 @@ trait Delay {
 
 
 trait DelayedDistribution extends Delay {
-  def distributedFuture[T](mean: Double,
-                           stdev: Double,
-                           dist: Distribution[Double]
-                          ): Distribution[Double] =
-    dist.map(_*stdev + mean)
-}
\ No newline at end of file
+  def normalDistribution(mean: Double,
+                            stdev: Double,
+                            dist: Distribution[Double]
+                           ): Distribution[Double] =
+    dist.map(_ * stdev + mean)
+
+  def gammaDistribution(k: Double, theta: Double): Distribution[Double] = {
+    Distribution.gamma(k, theta)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/scala-academy/Castalia/file/1993066085/issues/source?bid=2998213&fileBranchId=3112713#l31)